packages/table/src/layers/regions.tsx

```diff
@@ -21,13 +21,26 @@ export interface IRegionLayerProps extends IProps {
     regions?: IRegion[];
 
     /**
-     * A callback interface for applying CSS styles to the regions.
+     * The array of CSS styles to apply to each region.
```

Comment:
how do we know which styles belong to which region? just by index? worth noting in docs here i think.